README.md

```diff
@@ -237,61 +239,46 @@ the output to a log file or other log management system.  See the
 [signalfx-agent command](./docs/signalfx-agent.1.man) doc for more information on
 supported command flags.
 
-#### Windows Standalone .zip
-If you don't want to use the installer script, we offer a
-.zip that can be deployed to the target host.  This bundle is available for
-download on the [Github Releases
-Page](https://github.com/signalfx/signalfx-agent/releases) for each new
-release.
-
-Before proceeding make sure the following requirements are met.
-- [.Net Framework 3.5](https://docs.microsoft.com/en-us/dotnet/framework/install/dotnet-35-windows-10) (Windows 8+)
-- [Visual C++ Compiler for Python 2.7](https://www.microsoft.com/EN-US/DOWNLOAD/DETAILS.ASPX?ID=44266)
-
-To use the bundle:
+#### Windows Chocolatey Package
+_Only available for Smart Agent v5.3.0 or later._
 
-1) Unzip it to a directory of your choice on the target system.
+If [Chocolatey](https://chocolatey.org/) is installed on your Windows system, you
```

Comment:
Do you think users will read this section if they _don't_ have Chocolatey installed? I think you can drop this entire sentence and replace it with
"To install the Smart Agent using Chocolatey, run the following PowerShell command as Administrator". The Microsoft name for it is **PowerShell**. Please use that whenever you refer to it.